test/benchmarks/benchmark_main.cc

```diff
@@ -7,13 +7,17 @@
 
 #include <benchmark/benchmark.h>
 
+#ifndef BUILD_ALT
```

Comment:
Instead of `BUILD_ALT` couldn't we use `ZLIB_COMPAT` instead?

Reply:
Could we?  I think I'm letting cmake use our headers in both circumstances, I'm only modifying what's being linked to.  I think the headers should be API compatible so that shouldn't be an issue.

Reply:
Yes it should work so long as you always ensure `ZLIB_COMPAT` is set..

Reply:
That ifdef is made to prevent a call to a cpu_feature_check() function that doesn't exist in vanilla zlib or other various forks.  If we're building against the same headers at compile time and only switching out what's being linked, then we end up with an undefined reference when linking to the alternative static library.